### DISCUSSION TITLE
API Key配置后回答出错，账号异常？

### DISCUSSION THREAD

**Discussion by youbaowangsheng** [ChatGPTNextWeb/NextChat#562] [Q&A] [answered]
chatGPT可以网页访问。
环境变量也正常配置了API Key。
但访问vercel上托管的网站，总是出现这个回答：
{
    "error": {
        "message": "Your access was terminated due to violation of our policies, please check your email for more information. If you believe this is in error and would like to appeal, please contact support@openai.com.",
        "type": "access_terminated",
        "param": null,
        "code": null
    }
}

**Comment by tjefferson** [comment, marked as answer]
你这是问了什么不得了的东西，你的key被openai禁用了。
